README.md

```diff
@@ -0,0 +1,33 @@
+# Stack Challenge
+The challenge: by implementing a simple `int` stack in Java, demonstrate the ability (a) to learn and use build automation tools (gradle), and (b) employ Java object-oriented programming fundamentals.
+
+## Goals
+1. Use gradle to manage dependencies, build, and test your project
+2. Implement the `shopstyle.example.IntStack` interface
+3. Verify that all tests in `shopstyle.example.IntStackImplTest` pass
+
+## Requirements
+1. Your stack implementation must have a constructor that accepts an initial size for the stack.
+2. The implementation must also have a zero-argument default constructor that initializes the stack with a size of your choice.
+3. The default initial size is up to you. However, this value must be set before runtime and must not be alterable, even by subclasses.
+
+## Using gradle
+[gradle](https://gradle.org) is a build automation tool. A full description of gradle's capabilities is outside our scope, here.
+
+For this project, you will use gradle to manage dependencies (specifically, JUnit), compile the project, and run the tests.
+
+To get started:
+
+1. Download and install gradle using the instructions [here](https://docs.gradle.org/current/userguide/installation.html)
```

Comment:
This isn't correct. I've set up the project with gradle wrapper (gradlew). This packages the build tool inside the repo so there is nothing to install. I will merge and update the instructions accordingly.